docs/how-to/ocm-control-plane-walkthrough.md

```diff
@@ -136,18 +149,29 @@ Open three windows, which we'll refer to throughout this walkthrough as:
 
 ### Start the Gateway Controller
 
-1. In `T1` run the following to start the Gateway Controller:
+1. In `T1` run the following to build and deploy the Gateway Controller in a container:
 
     ```bash
     make build-controller kind-load-controller deploy-controller
     ```
 
+    *Alternatively, in a new window we'll call `T3`, you can run the controller locally instead by running:*
+
+    ```bash
+    (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
+    ```
+
```

Comment:
I'm not sure if you should say this here, depends who this walkthough is targeted at i guess, but running the controller like that is really for devs i would say and if you are just trying it out you should deploy it.

It also turns out people just blindly copy all these things and paste away into terminals so you are inviting people into a situation where they have the controller running in two places which wouldn't be ideal.

Reply:
that's a good point actually. We could do with a decent dev setup guide which is quite different to a "here's how to use this thing" guide.